pages/library/[libGameId]/index.jsx

```diff
@@ -0,0 +1,95 @@
+import styled from "styled-components";
+import BackButton from "@/components/buttons/BackButton";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import Image from "next/image";
+
+const fetcher = (...args) => fetch(...args).then((response) => response.json());
+
+export default function LibraryGameDetails() {
+  const router = useRouter();
+
+  const { data, error, isLoading } = useSWR(
+    `/api/details/${router.query.libGameId}`,
+    fetcher
+  );
+
+  if (isLoading) return <div>Loading...</div>;
+
+  if (error) return <div>Error: {error.message}</div>;
+
+  return (
+    <>
+      <StyledContainer>
+        <StyledBackgroundImage
+          src={`https://images.igdb.com/igdb/image/upload/t_screenshot_med/${data[0].artworks[0].image_id}.jpg`}
+          width={data[0].artworks[0].width}
+          height={data[0].artworks[0].height}
+          alt={`artwork of ${data[0].name}`}
+        ></StyledBackgroundImage>
```

Comment:
This could also be written as a self-closing element:

```suggestion
        />
```